logchange-core/src/main/java/dev/logchange/core/infrastructure/persistance/changelog/FileChangelogRepository.java

```diff
@@ -50,6 +52,25 @@ public Changelog find() {
         return Changelog.of(versions, archives);
     }
 
+    @Override
+    public Changelog findXML() {
```

Comment:
DO we need it?

Reply:
For now we don't, however I added it if, down the line, there was a need to implement reading changes.xml archives (like you currently do with markdown)

Reply:
you mean like in case of merging current changes.xml (in project) file with file generated by this plugin?

Reply:
As in, someone has a changes.xml from a previous project, and they want to merge it into the newly generated one by the plugin